### PR TITLE
docs: update digital shadow documentation

### LINKED ISSUE #501: Update Digital Shadow Documentation + Openapi spec for get uniqueProductsWithTags
**Todo**
- Update Digital shadow (#297) documentation on main page to latest implementation
- add openapi spec for uniqueProductsWithTags (factoryinsight api for digital shadow)

Correct open comments from #431

### PR DESCRIPTION
**Pull request by JeremyTheocharis**
Fixes #501

### CODE REVIEW

**Comment by JeremyTheocharis** [comment]
@RobertFeldmannMck You are working on using Digital Shadow to integrate Quality Labs, right? I am currently reworking the official guide and it might help your understanding: https://deploy-preview-714--umh-docs.netlify.app/docs/concepts/digitalshadow/

File: docs/content/en/docs/Concepts/mqtt/index.md [hunk 1]
```diff
@@ -539,11 +622,26 @@ Shopfloor insights are recommendations for action that require concrete and rapi
...
+## Explanation of IDs
+
+There are various different IDs that you can find in the MQTT messages. This section is designed to give an overview.
+
+
+| Name | data type/format | description |
+|-----|-----|--------------------------|
+| [`product_id`](/docs/concepts/mqtt/#explanation-of-ids) | int | The type of product that should be produced in an order for a specific asset. Can be used to retrieve the target speed. |
+| [`order_id`](/docs/concepts/mqtt/#explanation-of-ids) | int | Order ID, which provides the type of product (see [`product_id`](/docs/concepts/mqtt/#explanation-of-ids)) and the amount of pieces that should be produced. |
+| [`uniqeProductAlternativeID`](/docs/concepts/mqtt/#explanation-of-ids) | int | In short: AID. Used to describe a single product of the type [`product_id`](/docs/concepts/mqtt/#explanation-of-ids) in the order [`order_id`](/docs/concepts/mqtt/#explanation-of-ids). This is the ID that might be written on a physical on the product and is usually the relevant ID for engineers and for production planning. It usually stays the same. |
```
Comment (larswitte) [inline docs/content/en/docs/Concepts/mqtt/index.md:634]: This is the ID that might be written on the product (with a physical Label, lasered on to it etc.: technologie independant) and is usually the relevant...

**Review by larswitte — CHANGES_REQUESTED** [review]
Thank you Jeremy, I like the new structure a lot.
I only found small spelling stuff.

**Comment by JeremyTheocharis** [comment]
Ready for review. I've created #718 to consolidate everything as it does not make sense to work on this PR anymore. Nevertheless, this PR improved the current state and therefore should be merged.

**Review by Scarjit — APPROVED** [review]

**Pull request merged**